invenio_app_ils/documents/jsonresolvers/document_relations.py

```diff
@@ -26,7 +26,7 @@ def jsonresolver_loader(url_map):
     def relations_resolver(document_pid):
         """Resolve record relations and add metadata."""
         document = Document.get_record_by_pid(document_pid)
-        return document.relations
+        return document.relations(sort_by = ["edition"])
```

Comment:
Question: I guess that `sort_by` is a list because we want to be able to define a different sort type for the different type of relations?

Reply:
Yes, as relations have a separate object list in the response